x/liquidstakeibc/keeper/setup_suite_test.go

```diff
@@ -288,3 +284,219 @@ func (suite *IntegrationTestSuite) SetupICAPath(path *ibctesting.Path, owner str
 
 	return nil
 }
+
+func (suite *IntegrationTestSuite) TestOneFullFlow() {
+	pstakeApp := suite.app
+	k := pstakeApp.LiquidStakeIBCKeeper
+	hc, ok := k.GetHostChain(suite.chainA.GetContext(), suite.chainB.ChainID)
+	suite.True(ok)
+	senderAcc := suite.chainA.SenderAccount
+	//user liquidstakes
+	msgLiquidStake := types.NewMsgLiquidStake(sdk.NewInt64Coin(hc.IBCDenom(), 1000000), senderAcc.GetAddress())
+	result, err := suite.app.MsgServiceRouter().Handler(msgLiquidStake)(suite.chainA.GetContext(), msgLiquidStake)
+	suite.NotNil(result)
+	suite.NoError(err)
+
+	// user redeems
+	msgRedeem := types.NewMsgRedeem(sdk.NewInt64Coin(hc.MintDenom(), 100000), senderAcc.GetAddress())
+	result, err = suite.app.MsgServiceRouter().Handler(msgRedeem)(suite.chainA.GetContext(), msgRedeem)
+	suite.NotNil(result)
+	suite.NoError(err)
```

Comment:
Maybe we should also check the amount of the staking account has the redeemed amount ?

Reply:
true, but i want to add it in individual unit tests, each ibc test has a setup code, now I can just copy from here, because got this to work